client/components/domains/domain-product-price/index.jsx

```diff
@@ -135,7 +135,7 @@ class DomainProductPrice extends React.Component {
 		return (
 			<div className={ className }>
 				<div className={ productPriceClassName }>
-					<span>{ translate( 'Free' ) }</span>
+					<span>{ translate( 'Free', { context: 'Adjective refers to subdomain' } ) }</span>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Free', { context: 'logged-out-homepage'} )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/emilyaudela-patch-2)